### PR TITLE
POST request for theme variable

### PR DESCRIPTION
**Pull request by aayusharora**

### CODE REVIEW

File: src/generator/www/js/main.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ $(document).ready(function() {
   let $email =$("#email");
   let $name  =$("#name"); 
   let $url  =$("#url");
+  let $theme =$("#theme");
```
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: ["$" is not defined.](https://www.codacy.com/app/triplez/open-event-webapp/file/2721312972/issues/source?bid=3233857&fileBranchId=3347760#l6)

Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Combine this with the previous 'let' statement.](https://www.codacy.com/app/triplez/open-event-webapp/file/2721312972/issues/source?bid=3233857&fileBranchId=3347760#l6)

File: src/generator/www/index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 </head>
 <body>
 <div class="container"></br></br>
-	<form class="col-md-offset-4 col-xs-offset-2 col-xs-8 col-md-4 form-group generator_form" name="form" method="post" role="form"  enctype="multipart/form-data" action="/generate">
```
Comment (championswimmer) [inline src/generator/www/index.html]: Till we are using the html submit action, we cannot remove action tag from here

File: src/generator/www/js/main.js (hunk 1 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Combine this with the previous 'let' statement.](https://www.codacy.com/app/triplez/open-event-webapp/file/2737543959/issues/source?bid=3233857&fileBranchId=3347760#l6)

Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: ["$" is not defined.](https://www.codacy.com/app/triplez/open-event-webapp/file/2737543959/issues/source?bid=3233857&fileBranchId=3347760#l6)

File: src/generator/www/index.html [hunk 3]
```diff
@@ -20,10 +20,10 @@
     	<input type="url" class="form-control" id="url" name="url">
  		</br>
   		<label for="theme">Choose your theme</label>
-  		<select class="form-control" id="theme" name="theme">
-    		<option value="1">Light</option>
-    		<option value="2">Dark</option>
-  		</select>
+  		  <select class="form-control" id="theme" name="theme">
+    		  <option value="1">Light</option>
```
Comment (championswimmer) [inline src/generator/www/index.html]: values should be changed to light and dark. there is no change from what already exists in master now.

File: src/generator/www/js/main.js (hunk 1 above)
Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: [Combine this with the previous 'let' statement.](https://www.codacy.com/app/triplez/open-event-webapp/file/2738101723/issues/source?bid=3233857&fileBranchId=3347760#l6)

Comment (juslee) [inline src/generator/www/js/main.js]: Codacy Issue found: ["$" is not defined.](https://www.codacy.com/app/triplez/open-event-webapp/file/2738101723/issues/source?bid=3233857&fileBranchId=3347760#l6)

**Comment by aayusharora** [comment]
@championswimmer, something bad happened to my master, that's why this is happening . Now, I think it's correct.

**Pull request merged**